src/interface/TWCoinTypeConfiguration.cpp

```diff
@@ -255,6 +260,7 @@ TWString *_Nonnull TWCoinTypeConfigurationGetID(enum TWCoinType type) {
     case TWCoinTypeTheta: string = "theta"; break;
     case TWCoinTypeLux: string = "lux"; break;
     case TWCoinTypeQtum: string = "qtum"; break;
+    case TWCoinTypeBravo: string = "bravocoin.com"; break;
```

Comment:
just `bravocoin`

Reply:
There are two bravocoin actually. bravocoin.io and bravocoin.com. To distinguish, I've used here bravocoin.com. Any suggestion?

Reply:
what's the difference?

Reply:
bravocoin.io owned by someone else.

Reply:
it seems bravocoin.io is an ERC20 token, so just use `bravocoin`

Reply:
Okay